### DISCUSSION TITLE
More ideas for OpenHV

### DISCUSSION THREAD

**Discussion by WilliamWeberBerrutti** [OpenHV/OpenHV#757] [Ideas]
I had some more ideas that I'd like to share:

1. What if the map was activated since start, but you need Radar Dome to see allied and enemy (not including self) units and buildings on it?

2. What about increasing LOS of every unit (and maybe buildings) slightly? Maybe +1 or +2? (this idea was already given before, but was more drastic on the LOS changes)

3. What about some new neutral captureable buildings (maybe all effects can be cumulative?):

- one that gives extra power percentage (maybe +5%?)
- one that increases the range of all turrets (maybe +1?)
- one that makes units cheaper (maybe 5%?)
- one that makes buildings cheaper (maybe 5%?)
- one that makes buildings construct faster (maybe 5%?)

**Comment by Dzierzan** [comment]
`1. What if the map was activated since start, but you need Radar Dome to see allied and enemy (not including self) units and buildings on it?`

Maybe. Modern RTS games give you radar out of the box. However, old KKND game had specific radar.

- No HQ upgrade - no radar
- +1 upgrade - radar but you only see your actors on minimap
- +2 upgrade - you also see ally and enemy actors on minimap

So having radar by default but seeing only your troops could work and it wouldn't take Radar Dome building purpose (it's still needed for other things anyway). Up to @Mailaender 

`2. What about increasing LOS of every unit (and maybe buildings) slightly? Maybe +1 or +2? (this idea was already given before, but was more drastic on the LOS changes)`

I think I might have made LOS changes too drastic. I could increase it a bit.

`3. What about some new neutral captureable buildings (maybe all effects can be cumulative?):`

IMO we've got enough tech buildings.

**Reply by Mailaender** [reply]
We have no reveal map / enemy units support power on purpose. I think scouting is an essential part of an RTS. Not something that should be obsolete in late game.

**Reply by Rampoina** [reply]
I think you misunderstood, nobody is talking about revealing the whole map or units.

**Reply by GraionDilach** [reply]
One of the problems wrt minimap is that it is exposed to the screen settings (especially with OpenHV's smaller cell sizes). The minimap is extremely useful when the UI takes away a major portion of the screen, but becomes less relevant when the sidebar only obscures say, less than 20% of the screen space.

RA2 mods giving out the minimap at the start was a counter step against this devaluation. I like the idea that the minimap would be enabled but only reveal terrain without the radar for this reason.
